models/property.py

```diff
@@ -61,4 +62,14 @@ def find_by_name(cls, name):
 
     @classmethod
     def find_by_manager(cls, manager_id):
-        return cls.query.filter_by(propertyManager=manager_id).all()
+        return cls.query.filter(cls.managers.any(UserModel.id == manager_id)).all()
+
+    @classmethod
+    def set_property_managers(cls, ids):
+        managers = []
+        if ids:
+            for id in ids:
+                user = UserModel.find_by_id(id)
+                if user:
+                    managers.append(user)
```

Comment:
What should happen if a user does not exist?

Reply:
Currently, we return an empty manager list if none of the id exist. When the fe ticket that list manager in the create new property page is completed, then the ids are pre-validated as existing managers.

Reply:
Thank you. I'm wondering what we should do if partial id's exist? Should we ignore the ones that don't and persist the ones that do, or raise an error?

Reply:
We can always address this later on too when we implement marshmallow for input validation.

Reply:
I thought about both cases as well. Only include property-manager role user and raise an error. I can do both here which might be redundant later on when fe only allow project manager role be selected when create/update property. We'll deal with that when we come to that road.

Reply:
That's true. Something to think about.